src/api/data/scheduledMessage/scheduledMessage.mutations.ts

```diff
@@ -6,9 +6,14 @@ import { subscriptions } from './scheduledMessage.subscriptions'
 import * as uuid from 'uuid'
 import * as moment from 'moment'
 
+// Test purposes
+import sendMail, { IMailerHTML } from '../../../config/mailer'
+import { templates } from '../../../config/mailer/templateParser'
+
 const Mutation = gql`
   extend type Mutation {
     createScheduledMessage(messageData: ScheduledMessageInput!): ScheduledMessage
+    reviewRequest(emailData: ReviewRequestData): Boolean
```

Comment:
Where is `reviewRequestData` coming from?

Reply:
jk, found it.

Reply:
the type is defined on `src/api/data/scheduledMessage/scheduledMessage.type.ts`